ecr/base.go

```diff
@@ -32,6 +32,15 @@ import (
 var (
 	errImageNotFound     = errors.New("ecr: image not found")
 	errGetImageUnhandled = errors.New("ecr: unable to get images")
+
+	// supportedImageMediaTypes lists supported content types for images.
+	supportedImageMediaTypes = []string{
+		images.MediaTypeDockerSchema1Manifest,
+		images.MediaTypeDockerSchema2Manifest,
+		images.MediaTypeDockerSchema2ManifestList,
+		ocispec.MediaTypeImageIndex,
+		ocispec.MediaTypeImageManifest,
```

Comment:
Can you order these in preference order?

```suggestion
		ocispec.MediaTypeImageIndex,
		ocispec.MediaTypeImageManifest,
		images.MediaTypeDockerSchema2Manifest,
		images.MediaTypeDockerSchema2ManifestList,
		images.MediaTypeDockerSchema1Manifest,
```

Reply:
Updated!